cypress/integration/checkout/checkout.basic.spec.js

```diff
@@ -0,0 +1,65 @@
+import { bigtestGlobals } from '@bigtest/globals';
+
+bigtestGlobals.defaultInteractorTimeout = 10000;
+
+describe('Check Out', () => {
+  let ITEM_BARCODE;
+
+  before(() => {
+    cy.visit('/');
+
+    cy.login('diku_admin', 'admin');
+  });
+
+  beforeEach(() => {
+    ITEM_BARCODE = Number(new Date()).toString();
+
+    cy.getToken('diku_admin', 'admin')
+      .then(() => {
+        cy.getLoanTypes({ limit: 1 });
+        cy.getMaterialTypes({ limit: 1 });
+        cy.getLocations({ limit: 1 });
+        cy.getHoldingTypes({ limit: 1 });
+        cy.getHoldingSources({ limit: 1 });
+        cy.getInstanceTypes({ limit: 1 });
+        cy.getUsers({ limit: 1, query: '"personal.firstName"="checkin-all" and "active"="true"' });
+      })
+      .then(() => {
+        cy.getUserServicePoints(Cypress.env('users')[0].id);
+
+        cy.createInstance({
+          instance: {
+            instanceTypeId: Cypress.env('instanceTypes')[0].id,
+            title: `Pre-checkout instance ${Number(new Date())}`,
+          },
+          holdings: [{
+            holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
+            permanentLocationId: Cypress.env('locations')[0].id,
+            sourceId: Cypress.env('holdingSources')[0].id,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+              materialType: { id: Cypress.env('materialTypes')[0].id },
+            }],
+          ],
+        });
+      });
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Basic flow', function () {
+    cy.visit('/checkout');
+
+    cy.checkOutItem(Cypress.env('users')[0].barcode, ITEM_BARCODE);
```

Comment:
What is the difference between this approach? Wondering whether one approach can be used.

https://github.com/folio-org/stripes-testing/pull/105/files#diff-ba340d01f94ccbd02897d172ef5abadd0dbd2b2c2e4e22cb44c9b43c5997d26eR53-R57

Overall I see the setup for check-in is almost the same, only the [query differs](https://github.com/folio-org/stripes-testing/pull/105/files#diff-ba340d01f94ccbd02897d172ef5abadd0dbd2b2c2e4e22cb44c9b43c5997d26eR25) and a [title](https://github.com/folio-org/stripes-testing/pull/105/files#diff-ba340d01f94ccbd02897d172ef5abadd0dbd2b2c2e4e22cb44c9b43c5997d26eR33) for the item. Wondering whether it makes sense to create a helper and use it in two places.

Reply:
> What is the difference between this approach? Wondering whether one approach can be used.

here we test checkout, in check in test we use API to create item with `Check out` status to avoid UI interactions - API is much faster that UI

> Overall I see the setup for check-in is almost the same
yeah, this PR to initialize work and provide a working example